packages/lumx-react/src/components/text/Text.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import { COMPONENT_PREFIX } from '@lumx/react/constants';
+import { EllipsisType, GenericProps, truncate } from '@lumx/react/utils';
+
+/**
+ * Defines the props of the component.
+ */
+interface TextProps extends GenericProps {
+    /** The text to display */
+    text: string;
+    /** The max number of characteres to display */
+    maxChars?: number;
+    /** Works with maxChars. The ellipsis method to use */
+    ellipsisType?: EllipsisType;
+}
+
+/**
+ * The display name of the component.
+ */
+const COMPONENT_NAME = `${COMPONENT_PREFIX}Text`;
+
+/**
+ * The default value of props.
+ */
+const DEFAULT_PROPS: Partial<TextProps> = {
+    maxChars: -1,
+    ellipsisType: EllipsisType.END,
```

Comment:
Generally, when we have optional props with _falsy_ values, we prefer not to set them.